src/Sylius/Component/Core/Repository/ProductRepositoryInterface.php

```diff
@@ -33,10 +33,12 @@ public function createQueryBuilderWithLocaleCodeAndTaxonId($localeCode, $taxonId
     /**
      * @param string $code
      * @param ChannelInterface $channel
+     * @param string $locale
      *
      * @return QueryBuilder
      */
-    public function createQueryBuilderForEnabledByTaxonCodeAndChannel($code, ChannelInterface $channel);
+    public function createQueryBuilderForEnabledByTaxonCodeAndChannel($code, ChannelInterface $channel, $locale);
```

Comment:
It's not consistent, but in most repository methods we use ``$locale``. I think it's clear enough, as in code it almost everywhere a locale _code_, not _name_, but we should obviously keep a consistency with this.